client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "namex",
-  "version": "2.1.8",
+  "version": "2.1.9",
```

Comment:
After you change the package.json file, you have to regenerate the package-lock.json file (and check it in).

The package.json file contains the spec/config for the app.

The package-lock.json file is the list of _actual_ packages being imported (including all dependencies). This needs to be checked is so that the build server uses _exactly_ the same packages that you used, and tested with, locally.

Just do 'npm install', and commit the lock file, too.

PS: Sometimes we cheat a bit and just manually update the app version in the package lock file, but it's better to actually "install" any changes.

PPS: We have to all sync up (including the build server) if we are going to update node/npm... so don't do it :)  Here are the current versions we use across all our UI projects:
```
severin@DESKTOP-2XZPXD3:~/repos/name-examination/client$ node --version
v10.19.0
severin@DESKTOP-2XZPXD3:~/repos/name-examination/client$ npm --version
6.14.4
```

Reply:
after i run npm install... do i need to run     npm audit fix    to try and reduce the number of vulnerabilities  or just leave it.

found 423 vulnerabilities

Reply:
done.